homeassistant/components/reolink/number.py

```diff
@@ -62,10 +63,111 @@ class ReolinkNumberEntityDescription(
         native_step=1,
         get_min_value=lambda api, ch: api.zoom_range(ch)["focus"]["pos"]["min"],
         get_max_value=lambda api, ch: api.zoom_range(ch)["focus"]["pos"]["max"],
-        supported=lambda api, ch: api.zoom_supported(ch),
+        supported=lambda api, ch: api.supported(ch, "zoom"),
         value=lambda api, ch: api.get_focus(ch),
         method=lambda api, ch, value: api.set_zoom(ch, int(value)),
     ),
+    ReolinkNumberEntityDescription(
+        key="floodlight_brightness",
+        name="Floodlight brightness",
+        icon="mdi:spotlight-beam",
+        entity_category=EntityCategory.CONFIG,
+        native_step=1,
+        get_min_value=lambda api, ch: 1,
+        get_max_value=lambda api, ch: 100,
```

Comment:
I guess we can just use the `native_min_value` and `native_max_value` in those cases, and omit the additional lambda.

Reply:
Thanks, done this everywhere where the lambda was not required.